README.md

```diff
@@ -387,8 +388,9 @@ Create an ansible inventory file named `hosts` **in your working directory**
 
 ```
 [openwisp2]
-192.168.56.2
+username@192.168.56.2
```

Comment:
this is not right

Reply:
It is correct if we have only one user on the remote host, but if we have multiple hosts then to use the `192.168.56.2` syntax in our inventory file, we will need to specify the SSH user in our Ansible playbook.